lib/hexpm_web/templates/blog/007-hex-v0.19-released.html.md

```diff
@@ -0,0 +1,13 @@
+## Hex v0.19 released
+
+<div class="subtitle">15 January, 2019 · by Eric Meadows-Jönsson</div>
+
+The v0.19 release includes an important security fix to anyone accessing Hex repositories through a mirror. A bug has been found that would allow a malicious mirror to serve modified versions of Hex packages. [hex](https://github.com/hexpm/hex) versions `0.14.0` to 0.18.2 and [rebar3](https://github.com/erlang/rebar3) versions `3.7.0` to `3.7.5`. Make sure to update to hex `0.19.0` and rebar3 `3.8.0`.
```

Comment:
should this have been:

> hex versions `0.14.0` to `0.18.2` and rebar3 versions `3.7.0` to `3.7.5` are vulnerable. Make sure to update to hex `0.19.0` and rebar3 `3.8.0`.

if so, happy to make the change on master.

Reply:
Yes, please change.